capitals.rb

```diff
@@ -151,3 +151,46 @@
     name: "Wyoming",
     capital: "Cheyenne"
 }]
+
+def game
+states = @states.shuffle
```

Comment:
To add the properties to each state...
```rb
states.each do |state|
  state[:correct_guesses] = 0
  state[:incorrect_guesses] = 0
end
```

This will set up the right info so that the program to track the number of times the user gets a certain question right and wrong.